### PR TITLE
	TASK: Styling improvements for popovers and help text buttons

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

**Comment by aertmann** [comment]
:+1:

**Comment by aertmann** [comment]
before:
image
after:
image

before:
image
after:
image

before:
image
after:
image

**Comment by aertmann** [comment]
@dimaip maybe you could check this one?

**Comment by kitsunet** [comment]
Looks good and the dependency in Flow is merged, so :+1: from me. Would merge tomorrow if no one else checks it out...

**Comment by dimaip** [comment]
Looks great, :+1: by reading, will be able to test tomorrow.

File: TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss [hunk 1]
```diff
@@ -803,6 +803,11 @@
 		.neos-popover {
 			width: auto;
 			height: auto;
+			border-color: $grayMedium;
+
+			.neos-popover-title {
```
Comment (dimaip) [inline TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss]: I know we currently don't have any hard rules about how we write CSS, but could we perhaps start to move in the direction of having less nesting in CSS and towards somethings BEM-like. E.g. what's the sense of putting `.neos-popover-title` inside `.neos-popover`, where it already has this namespace inside its name?

**Comment by dimaip** [comment]
Verified, indeed it looks much better like this, thanks for the improvement!

**Comment by skurfuerst** [comment]
Hey,

@dimaip I'd suggest that we merge the change as-is; as the CSS guidelines to me are currently a "sidetrack" to this issue :-)

All the best,
Sebastian

**Comment by dimaip** [comment]
Of course, it was just a side note about CSS, feel free to ignore it

**Comment by skurfuerst** [comment]
OK, merging with previous votes!

**Pull request merged**